client/css/style.css

```diff
@@ -608,6 +608,12 @@ button {
 	display: none;
 	font-style: normal;
 }
+#chat .msg .inline-channel {
+	cursor: pointer;
+}
+#chat .msg .inline-channel:hover {
+	opacity: .8;
```

Comment:
I noticed existing opacities are floating between `.6` and `.8` but here I'd prefer `.6` to be consistent with existing links on the chat window.